README.md

```diff
@@ -18,8 +18,9 @@ For more information, see the blog post [Nextcloud in Digital Imaging](https://n
 
 ### Features
 
-* **Viewer:** A DICOM viewer which displays images grouped by study and series in selected folders and allows to manipulate images with imaging tools
-* **DICOM Dump:** A list of DICOM attributes displayed on the sidebar with image thumbnail
+* **Viewer:** A DICOM viewer which displays images grouped by study and series in selected folders and allows to manipulate images with imaging tools.
+* **DICOM Dump:** A list of DICOM attributes displayed on the sidebar with image thumbnail (*Note:* The Nextcloud files app plugin "Comments" must be activated to use this).
```

Comment:
I can't reproduce any issue with sidebar after i disable 'Comments' app. Are you sure about that?

Reply:
@ayselafsar 
Yes, I can reproduce this:
![deactivated_comments](https://user-images.githubusercontent.com/45886938/50626123-b1f34a80-0f2c-11e9-8b6e-4b6c4aceb8ea.png)
I'm on NC 15.0.0 and comments 1.5.0. As the screenshot shows, I got the disc when deactivating the comments-plugin.